config/config.yaml

```diff
@@ -0,0 +1,10 @@
+debug_mode: True
+
+desks:
+  - id : 1
+    name: "Desk 1"
+    sensor_pin: 2
+
+  - id : 2
+    name: "Desk 2"
+    sensor_pin : 3
```

Comment:
This will probably have to change since we're gonna be using I2C right?

Reply:
Correct, we can probably figure it out and fix this when we receive the sensor and learn more about how we can get it to work with I2C. My guess is that this field changes to a hex address, but we will see in the near future

Reply:
I thought the server was delegating ids. Should the user be setting id's in the config? These ids are globally unique which is easy for the server but hard for humans to do.

Reply:
@DominikSchmidtlein We're talking about sensor_pin not id!

Reply:
(My bad, I just realized you weren't responding to Erick and my conversation, and you're talking about something else) 

Yes you're correct, the server is delegating the IDs, and the config file is basically the persistence of how the raspberry pi knows which desk belongs to which ID. I personally think we should not let the user set anything in the config file, and should be managed by a separate py script that the user can run to change user-configurable values (if that makes sense)

Reply:
If you guys prefer, we can store all user-configurable values in the YAML file, and then maybe have a protected SQLite db where the PI can store all persistence and configuration that is not user-configurable

Reply:
Good idea, I would say maybe later. Might be slightly over complicated at this point.